ab_tool/static/ab_tool/js/experiments_dashboard.js

```diff
@@ -1,17 +1,61 @@
 $(document).ready(function(){
 
-    //double click-submission protection
-    $('.submitter, form a, form [type="submit"], form button').click(function (event) {
-        if ($(this).hasClass("disabled")) {
+    // double click-submission protection for create/update button
+    $('.modal-form').submit(function (event) {
+        var $submit_button = $(this).find('.modal-submit');
+        if ($submit_button.hasClass("disabled")) {
             event.preventDefault();
         }
-        $(this).addClass("disabled");
+        else {
+            // disables create/update button, cancel link, and form inputs
+            $submit_button.addClass("disabled");
+            $submit_button.siblings('.modal-cancel').addClass("disabled");
+        }
     });
 
     function is_valid_url(url) {
+        //if URL input element is used, this should match the pattern attribute or the custom validation
         return /^(http(s)?:\/\/)?(www\.)?[a-z0-9]+([\-\.]{1}[a-z0-9]+)*\.[a-z]{2,5}(:[0-9]{1,5})?(\/.*)?$/i.test(url);
     }
 
+    function enable_submit_if_no_errors() {
```

Comment:
For this to work you need to add .clean to ALL input fields. How you have it now, you are able to submit the form by just filling out the Name of the intervention. As far as I know we don't want to allow creation of intervention points with empty URLs. We should talk about making this work better.